apps/src/templates/teacherDashboard/RecentCourses.story.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
Generalizing seems like a great idea if it's going to be basically the same across the scenarios. The approach that I generally take is that when I find myself rewriting a bunch of code, I look to generalize (i.e. if you get to your second or third component and find it has a lot of logic as your first component, that can be a good time to extra some stuff to be shared).